### PR TITLE
feat(condo): DOMA-6784 B2BAppPermission name field added

### PR DESCRIPTION
**Pull request by SavelevMatthew**
image

### CODE REVIEW

File: apps/condo/domains/miniapp/schema/B2BAppPermission.js [hunk 1]
```diff
@@ -58,7 +67,29 @@ const B2BAppPermission = new GQLListSchema('B2BAppPermission', {
...
+            hooks: {
+                resolveInput: ({ resolvedData, fieldPath }) => {
+                    if (fieldPath in resolvedData) {
+                        return resolvedData[fieldPath].trim()
+                    }
 
+                    return undefined
+                },
+                validateInput: ({ resolvedData, fieldPath, context }) => {
+                    if (resolvedData[fieldPath].length > MAX_PERMISSION_NAME_LENGTH) {
```
Comment (Alllex202) [inline apps/condo/domains/miniapp/schema/B2BAppPermission.js:87]: Is there a minimum length?

Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/schema/B2BAppPermission.js:87]: Nope, there's not.

I've named this field `description` previously, but then I thought that it was too unclear, so I used field `shortDescription`. 

Then I thought, what is short? So I've added a limit.

And just before pushing when I was looking at designs again I thought that `name` option is the clearest one. But left old limitation :)

Comment (Alllex202) [reply apps/condo/domains/miniapp/schema/B2BAppPermission.js:87]: That is, I can make an empty name? What will the user see in the interface?

File: apps/condo/domains/miniapp/schema/B2BAppPermission.js [hunk 2]
```diff
@@ -19,6 +23,11 @@ const ERRORS = {
         type: PERMISSION_KEY_WRONG_FORMAT_ERROR,
         message: 'Incorrect key format. The key must start with the prefix "can", have lowerCamelCase and answer the question: "what is allowed to the user with this key?". Example: canManagePasses, canReadConfig, etc.',
     },
+    TOO_LONG_SHORT_DESCRIPTION: {
```
Comment (Alllex202) [inline apps/condo/domains/miniapp/schema/B2BAppPermission.js:26]: Сan we name this error according to its type?)

File: apps/condo/domains/miniapp/schema/B2BAppPermission.js [hunk 3]
```diff
@@ -19,6 +23,11 @@ const ERRORS = {
         type: PERMISSION_KEY_WRONG_FORMAT_ERROR,
         message: 'Incorrect key format. The key must start with the prefix "can", have lowerCamelCase and answer the question: "what is allowed to the user with this key?". Example: canManagePasses, canReadConfig, etc.',
```
Comment (AntonAL) [inline apps/condo/domains/miniapp/schema/B2BAppPermission.js:24]: There is a `canManagePasses` field for example values.
ADR: https://confluence.in.doma.ai/pages/viewpage.action?pageId=967573505

```suggestion
        message: 'Incorrect key format. The key must start with the prefix "can", have lowerCamelCase and answer the question: "what is allowed to the user with this key?"',
        correctExample: 'canManagePasses'
```

Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/schema/B2BAppPermission.js:24]: `correctExample` is too strange. I have not seen it in any API I've worked with

File: apps/condo/domains/miniapp/schema/B2BAppPermission.js (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/schema/B2BAppPermission.js]: Refactored validation :)

**Review by Alllex202 — APPROVED** [review]

**Pull request merged**